Libplanet.Tests/Action/ActionEvaluatorTest.cs

```diff
@@ -258,6 +258,31 @@ DumbAction MakeAction(Address address, char identifier, Address? transferTo = nu
                         : null);
             }
 
+            var totalSupplyGetterGetter = new Func<
```

Comment:
Not that I have any better suggestion at the moment, but the name suggests bad code smell (probably bad `ActionEvaluator` design). 😅

Reply:
Actually this function wasn't really needed as it could have been just put in two different places, but it would cause the 300 line limit warning to trigger. :P Maybe it might be better to have something similar to NullTotalSupplyGetter.

Reply:
Renamed to `totalSupplyGetterFromDict`.